styles/place.css

```diff
@@ -0,0 +1,123 @@
+:root {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    --textcolor: black;
+    --backgroundcolor: rgb(12, 196, 12);
+}
+
+body {
+    font-family: "Roboto", serif;
+    color: var(--textcolor);
```

Comment:
I suggest adding `margin: 0;` to the `body` to ensure there’s no white border around all the content.